src/traffic-collector/azext_traffic_collector/aaz/latest/network_function/traffic_collector/_create.py

```diff
@@ -0,0 +1,515 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "network-function traffic-collector create",
+)
+class Create(AAZCommand):
+    """Creates a Azure Traffic Collector resource
+
+    :example: Create a new traffic collector
+        az az network-function traffic-collector create --resource-group rg1 --azure-traffic-collector-name atc1 --location eastus
+    """
+
+    _aaz_info = {
+        "version": "2022-08-01",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.networkfunction/azuretrafficcollectors/{}", "2022-08-01"],
+        ]
+    }
+
+    AZ_SUPPORT_NO_WAIT = True
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        return self.build_lro_poller(self._execute_operations, self._output)
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.azure_traffic_collector_name = AAZStrArg(
+            options=["-n", "--name", "--azure-traffic-collector-name"],
```

Comment:
usually, we will not add azure in argument name. Recommend renaming it to `--traffic-collector-name`